webui/src/lib/components/auth/layout.jsx

```diff
@@ -8,9 +8,11 @@ import Card from "react-bootstrap/Card";
 
 import Layout from "../layout";
 import {Link} from "../nav";
+import {useLoginConfigContext} from "../../hooks/conf";
 
 
 export const AuthLayout = ({ children, activeTab }) => {
+    const simplified = useLoginConfigContext()?.RBAC === 'simplified';
```

Comment:
Sorry, I don't understand.  `useLoginConfigContext` returns a `LoginConfig`.  I extract RBAC (a string) from it and perform a computation to get a boolean.  This occurs on each render -- whenever the AuthLayout function runs.  I could say
```suggestion
    const { RBAC: rbac } = useLoginConfigContext() /* modified always to return an object */; 
    const simplified = rbac === 'simplified';
```
but I don't understand when there would ever be a different between the two, functional or even nonfunctional.

Reply:
Maybe I didn't explain myself clearly. I'll try again, and, hopefully, do a better job of it :)
It's not that what you're doing is wrong, and you're correct that they are functionally identical. In your case, you're accessing a specific property, so you aren't going to have any issues. However, the idiomatic syntax for using the return value of hooks is to destructure the result.
Why? Take this code as an example:
```JavaScript
const someObject = useSomeHook();

useEffect(() => {
  if (someObject.someProperty) {
    // do something...
  }
}, [someObject]);
```
In this case, `someProperty` can change without the ref of `someObject` changing. I've been asked for assistance twice in the last couple of months because of something like this, so it's obviously not so obvious to everyone :)
So it's not needed, but it's recommended because it makes it easier for _others_ to avoid this mistake. 
All that being said, I understand this isn't the same case as the one I'm talking about, so I leave it up to you if that's something you want to change.